### DISCUSSION TITLE
Version 6.x cannot parse four digit version tags, but v5.12 can

### DISCUSSION THREAD

**Discussion by ITaluone** [GitTools/GitVersion#4287] [Q&A] [answered]
Hi

Maybe I am doing something wrong, but after upgrading to GitVersion >=6 my version tags (which consists of four parts) cannot be parsed anymore (meaning are completely ignored).

I have an app which on the time of starting the app had three version parts `x.y.z`, but I changed that to `x.y.z.build`.

### Setup
My git history has this type of tags
- 1.0.0
- 1.1.0
- 1.2.0.1234
- 1.3.0.2345

Now, if I run `dotnet gitversion` with version 5.12.0 I correctly get as next version `1.4.0` i.e. ignoring the fourth part.
But if I run this after upgrading to v6 (and update the `GitVersion.yml` accordingly) I get `1.2.0` as next version.

I'm pretty sure I missed a configuration option, but cannot find it.

Any help really appreciated.

Thanks in advance

**Comment by arturcic** [comment, marked as answer]
https://gitversion.net/docs/reference/configuration#semantic-version-format you need to set the 
`semantic-version-format: Loose` in GitVersion.yml

**Reply by ITaluone** [reply]
Ah... thank you very much 🎉

I was sure that I missed something ;)

**Reply by ITaluone** [reply]
But: I don't read about this in the [docs](https://github.com/GitTools/GitVersion/blob/main/BREAKING_CHANGES.md) 🤔 
I would consider this as breaking change, too.

**Reply by arturcic** [reply]
> But: I don't read about this in the [docs](https://github.com/GitTools/GitVersion/blob/main/BREAKING_CHANGES.md) 🤔 
> I would consider this as breaking change, too.
> 

A PR that can fix this missing piece is very welcome, so that the next person doesn't spend the time as you did

**Reply by ITaluone** [reply]
Sure :)
